### DISCUSSION TITLE
Add Troubleshooting/Log explanation to docs

### DISCUSSION THREAD

**Discussion by jgilfoil** [renovatebot/renovate#11892] [Suggest an Idea]
Hey :wave: . As a new user to the product, it took me a long while to locate where logs for the hosted github app are get stored. I didn't notice the link off of PR's and so I was searching documentation, but all it really mentions about logs is the self-hosted version. 

I eventually found them only because of this post https://github.com/renovatebot/renovate/discussions/8800#discussioncomment-392504 . **It would be great if the log location and a troubleshooting section in general, existed in the https://docs.renovatebot.com/** pages. 

Thanks for the great product!

**Comment by rarkins** [comment]
The source for those pages are all hosted in this repo, under docs/usage/. Where specifically would you suggest adding it?

**Reply by HonkingGoose** [reply]
How about this:

- Create new page "Troubleshooting" in the Getting Started sidebar category, and cover:
  - Where to find the logs on hosted app.
  - Where to find logs if self-hosting.
  - How to find the relevant part of the logs.
  - How to get help (probably link to Renovate discussion).
- Add new entry in FAQ on troubleshooting that links to the new troubleshooting page

We could add some common questions/confusions in that troubleshooting section as well.

**Reply by HonkingGoose** [reply]
I'm working on the new troubleshooting documentation, see issue:

- #11896.

**Comment by MPV** [comment]
Here's another idea on how to also alleviate this issue:
- https://github.com/renovatebot/renovate/discussions/27175
